netdisco/discoverables/yeelight.py

```diff
@@ -0,0 +1,18 @@
+"""Discover Yeelight bulbs, based on Kodi discoverable."""
+from . import MDNSDiscoverable
+
+
+# pylint: disable=too-few-public-methods
+class Discoverable(MDNSDiscoverable):
+    """Add support for discovering Yeelight."""
+
+    def __init__(self, nd):
+        """Initialize the Yeelight discovery."""
+        super(Discoverable, self).__init__(nd, '_miio._udp.local.')
+
+    def info_from_entry(self, entry):
+        """Return most important info from mDNS entries."""
+        return (self.ip_from_host(entry.server), entry.port)
```

Comment:
I want to migrate netdisco from tuples to returning dicts of info so we can add extra info in the future without making breaking changes. Can you convert this to be a dict?